querysync/javatests/com/google/idea/blaze/qsync/testdata/deppathkinds/BUILD

```diff
@@ -1,6 +1,6 @@
-load("@rules_java//java:defs.bzl", "java_library")
-load("@protobuf//bazel:proto_library.bzl", "proto_library")
 load("@protobuf//bazel:java_proto_library.bzl", "java_proto_library")
+load("@protobuf//bazel:proto_library.bzl", "proto_library")
+load("@rules_java//java:defs.bzl", "java_library")
```

Comment:
Just a question, why are there two different kinds of loads: 
1. `load("@rules_java//java:defs.bzl", "java_library")`
2. `load("@rules_java//java:java_library.bzl", "java_library")`

Reply:
`defs.bzl` is older (and legacy). At the moment we're advising everybody to use `<symbol>.bzl`. This results in minimal loads (defs.bzl needs to load everything, and by accident there java_proto_library in rules_java, that pulls down protobuf).

If there's a preexisting use of `defs.bzl`, buildifier won't touch it.